File: pkg/models/card.go
package models

import (
	"database/sql"
	"time"
)

type Card struct {
	ID            int
	DeckID        int
	English       string
	Russian       string
	Association   sql.NullString
	Example       sql.NullString
	Transcription sql.NullString
	Image         sql.NullString
	ImageURL      sql.NullString
	Audio         sql.NullString
	AudioURL      sql.NullString
	CreatedAt     time.Time
}

File: web/src/models/card.ts
export interface AddCard {
  english: string;
  russian: string;
  association?: string;
  example?: string;
  transcription?: string;
  image?: File;
  audio?: File;
  image_url?: string;
  audio_url?: string;
}

export interface Card {
  id: number;
  deck_id: number;
  english: string;
  russian: string;
  association?: string;
  example?: string;
  transcription?: string;
  image?: File;
  audio?: File;
  image_url?: string;
  audio_url?: string;
	created_at?: string;
}
